<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mikhails-MacBook-Pro.local" name="Command line test" tests="7" failures="1" timestamp="4 Aug 2016 23:09:30 GMT" time="23.378" errors="0">
  <testcase name="californiaTotalsTest" time="2.607" classname="artID.FurstTest"/>
  <testcase name="mnTestTotal" time="2.337" classname="artID.FurstTest"/>
  <testcase name="nyTestTotal" time="2.142" classname="artID.FurstTest"/>
  <testcase name="otherStateTestTotal" time="2.022" classname="artID.FurstTest"/>
  <testcase name="titleAssetion" time="0.041" classname="artID.FurstTest"/>
  <testcase name="validateCheckoutButtonWithEmptyFields" time="0.024" classname="artID.FurstTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at artID.FurstTest.validateCheckoutButtonWithEmptyFields(FurstTest.java:30)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validateCheckoutButtonWithEmptyFields -->
  <testcase name="validateMainItems" time="0.13" classname="artID.FurstTest"/>
</testsuite> <!-- Command line test -->
